CMakeLists.txt

```diff
@@ -69,3 +69,5 @@ add_subdirectory(libtempered)
 add_subdirectory(libtempered-util)
 add_subdirectory(utils)
 add_subdirectory(examples)
+
+find_package(rock)
```

Comment:
So, [this line](https://github.com/rock-core/rock-package_set/blob/master/patches/visp.patch#L7) on control/visp patch is unnecessary, right?